proxy/proxy.go

```diff
@@ -1117,7 +1117,9 @@ func (p *Proxy) serveResponse(ctx *context) {
 	}
 
 	ctx.responseWriter.WriteHeader(ctx.response.StatusCode)
-	err := copyStream(ctx.responseWriter.(flusherWriter), ctx.response.Body, p.tracing, ctx.proxySpan)
+	fw := ctx.responseWriter.(flusherWriter)
```

Comment:
@szuecs Would it make sense to change the interface of [responseWriter in context](https://github.com/zalando/skipper/blob/3c9d0559f917ca7e35057103e367b38dc2dc47e5/proxy/context.go#L20) to something that extends `http.ResponseWriter` with the `Flush()` function? It should still satisfy everything that expects a plain `http.ResponseWriter`, shouldn't it?

Reply:
Should this be a new public interface or not?

Reply:
no, not a public. I think flushing should be controlled only in the proxy package.

Reply:
Do you think the last commited change makes sense?

Reply:
Yes, this is what I meant. A ResponseWriter that can also Flush. But you should judge if it makes sense and how it effects other parts of skipper.